src/index.scss

```diff
@@ -170,8 +170,15 @@ html, body, #root, #app, .full-height {
 	height: 100%;
 	display: flex;
 	flex-direction: column;
+	
 	@include desktop {
-		min-height: 775px;
+		min-height: 750px;
+	}
+	@include tablet {
+		height: calc(100% - 74.5px);
```

Comment:
We should probably avoid calculations like this for layout generally, but considering how the mobile/desktop navbars interact (toggled HTML) it doens't seem easy to convert this layout to one without `calc`.
Since we're about to rewrite this entire site anyways, I'll just approve this PR 👍